utils/utilityML.py

```diff
@@ -0,0 +1,175 @@
+import numpy as np
```

Comment:
I don't like simply copy and pasting a file like this. I suggest removing this file and updating the nominal file to be flexible.